.github/workflows/release.yml

```diff
@@ -27,13 +27,13 @@ jobs:
     - name: Install Knope
       uses: knope-dev/action@v1
       with:
-        version: 0.7.1 # Test before updating, breaking changes likely: https://github.com/knope-dev/action#install-latest-version
+        version: 0.16.0 # Test before updating, breaking changes likely: https://github.com/knope-dev/action#install-latest-version
```

Comment:
This is the version I have locally installed, and I've verified that `... --dry-run --verbose` behaves as expected in this repo on this branch.